contracts/SportMarkets/SportsAMM.sol

```diff
@@ -96,20 +96,20 @@ contract SportsAMM is Initializable, ProxyOwned, PausableUpgradeable, ProxyReent
     /// @return Curve usage is enabled?
     bool public curveOnrampEnabled;
 
+    /// @return maximum supported discount in percentage on sUSD purchases with different collaterals
+    uint public maxAllowedPegSlippagePercentage;
+
     /// @return Referrals contract address
     address public referrals;
 
     /// @return Default referrer fee
     uint public referrerFee;
 
-    /// @return The address of Parlay AMM
-    address public parlayAMM;
-
     /// @return The address of Apex Consumer
     address public apexConsumer;
 
-    /// @return maximum supported discount in percentage on sUSD purchases with different collaterals
-    uint public maxAllowedPegSlippagePercentage;
+    /// @return The address of Parlay AMM
+    address public parlayAMM;
```

Comment:
don't change order of properties due to storage :)

i know it is different on `opt-goerli` and `opt-main` so only change order locally without pushing